GliaWidgets/Sources/ViewModel/EngagementViewModel.swift

```diff
@@ -34,6 +36,7 @@ class EngagementViewModel: CommonEngagementModel {
     }
 
     deinit {
+        print("☠️", Self.self, ObjectIdentifier(self))
```

Comment:
Is this still relevant?

Reply:
Oops)

Reply:
Fixed 2095a493dd034ee78bb1220038e6b0b6fee41f0e